### ISSUE TITLE
variable units in make*Stat

### ISSUE THREAD

**Issue by ktoddbrown** [JGCRI/RCMIP5#113]
Variable units should either change or get set to NULL after make*Stat functions.

**Comment by bpbond** [comment]
Agreed. What *should* happen? Change to "Global mean of $units", for example?
